kong/core/reports.lua

```diff
@@ -99,8 +99,6 @@ local function send_report(signal_type, t, host, port)
 
   -- concat and send buffer
 
-  --print(concat(_buffer, ";", 1, mutable_idx))
```

Comment:
The same instruction `concat(_buffer, ";", 1, mutable_idx)` is being used in the line below. I believe it's easy to copy-paste it in case debugging is necessary.